### PR TITLE
[NFT-828] chore: config update and cleanup

### PR DESCRIPTION
**Pull request by cnasc**
- did a cleanup of the config file, removing fields we do not use
- found some fields that were used by files that never get referenced, so also deleted those files
- renamed `paprMemeSubgraph` to `paprSubgraph` to avoid ambiguity

### CODE REVIEW

File: hooks/useActivity/useActivity.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import { useConfig } from 'hooks/useConfig';
 import { useMemo } from 'react';
 import {
-  ActivityByControllerDocument,
-  ActivityByControllerQuery,
+  ActivityByControllerAndAccountDocument,
```
Comment (cnasc) [inline hooks/useActivity/useActivity.ts:4]: just sorting, autofix

File: hooks/useControllerPricesData/useControllerPricesData.ts [hunk 2]
```diff
@@ -146,20 +146,20 @@ function targets(
 ) {
   const now = Math.floor(Date.now() / 1000);
   const sortedTargets = [...targetUpdates].sort(
-    (a, b) => parseInt(a.timestamp) - parseInt(b.timestamp),
+    (a, b) => a.timestamp - b.timestamp,
```
Comment (cnasc) [inline hooks/useControllerPricesData/useControllerPricesData.ts:149]: after using new subgraph got a type error here. These are now recognized as numbers and shouldn't be parsed.

File: codegen/erc721.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 overwrite: true
-schema: 'https://api.thegraph.com/subgraphs/name/adamgobes/erc721-goerli'
+schema: 'https://api.thegraph.com/subgraphs/name/sunguru98/mainnet-erc721-subgraph'
```
Comment (wilsoncusack) [inline codegen/erc721.yml:2]: does it matter which we use 🤔

File: hooks/useControllerPricesData/useControllerPricesData.ts (hunk 2 above)
Comment (wilsoncusack) [reply hooks/useControllerPricesData/useControllerPricesData.ts:149]: yup! good catch

**Review by wilsoncusack — APPROVED** [review]
lfg!

**Pull request merged**